bokehjs/src/coffee/models/renderers/graph_renderer.coffee

```diff
@@ -1,72 +1,86 @@
-import {Circle} from "../glyphs/circle"
-import {MultiLine} from "../glyphs/multi_line"
-import {GraphSource} from "../graphs/graph_source"
 import {Renderer, RendererView} from "../renderers/renderer"
 
 import * as p from "core/properties"
-import {range} from "core/util/array"
+import {build_views} from "core/build_views"
 
 export class GraphRendererView extends RendererView
 
   initialize: (options) ->
     super(options)
-    @edges = @build_glyph_view(@model.edges)
-    @nodes = @build_glyph_view(@model.nodes)
 
-    @xscale = @plot_view.frame.xscales[@model.x_range_name]
-    @yscale = @plot_view.frame.yscales[@model.y_range_name]
+    @xscale = @plot_view.frame.xscales["default"]
+    @yscale = @plot_view.frame.yscales["default"]
+
+    @_renderer_views = {}
+    @node_view = build_views(@_renderer_views, [@model.node_renderer,], @plot_view.view_options())[0]
+    @edge_view = build_views(@_renderer_views, [@model.edge_renderer,], @plot_view.view_options())[0]
```

Comment:
not important now but maybe we could have a `build_view` and then `buid_views` that uses it in a loop to avoid this `[0]` indexing